### ISSUE TITLE
One must restart sagetv server after installing phoenix before selecting the STV for an HD300 client

### ISSUE THREAD

**Issue by boc-tothefuture** [stuckless/sagetv-phoenix-core#5]
On a new docker install of SageTV, if you install the phoenix plugin and then try and switch to the STV on an extender like the HD300 you received a garbled screen that is unreadable.  Rebooting the HD300 does not resolve the issue.  

Once the container is restarted the interface properly functions on the HD300.

**Comment by stuckless** [comment]
Thanks for adding this... I've run into this before as well... Typically after installing Phoenix (core apis) the sagetv server needs a reboot.  There is lot of stuff that Phoenix does and some of it requires that it be restarted.  I think that it would be good to review the phoenix initialization process, but since this only happens when you first installs phoenix, then, doing a restart is "quicker" (should add some notes around it, at th every least).

**Comment by boc-tothefuture** [comment]
@stuckless I agree - perhaps it was in the readme notes when I installed the plugin and I missed it.  We could potentially resolve this issue with updating the text describing the plugin?  Just noting that it needs a restart of the sage process to finalize installation.

It has been a long time since I developed the TVBrowser plugin..  So I don't recall.  There is no way to request a server restart at the completion of a plugin install is there?

**Comment by jusjoken** [comment]
I think the core should offer to load the new STV when installed and when an stv is switches to the core should force a restart like other library type installs.
